src/Board.py

```diff
@@ -52,6 +58,18 @@ def __init__(self, style: str = "default", size: int = 5) -> None:
         else:
             raise ValueError("Unrecognised map style")
 
+    def start(self, terrain_filter: Terrain | None = None):
+        return_list = []
+        for row in range(0, len(self.data)):
+            for spot in range(0, len(self.data[row])):
+                tile = (spot, row)
+                tile_object = self.get(tile)
+                if tile_object.terrain == terrain_filter or (
+                    terrain_filter is None and tile_object.terrain != Terrain.EMPTY
+                ):
+                    return_list.append(tile)
```

Comment:
I think it might be more helpful if we return the `tile_object` here.. what do you reckon? Then it's easier to call `.build` or whatever else on the tile without needing another call to get the object from the coords